my-react-app/src/app/screens/Login/utils.js

```diff
@@ -0,0 +1,3 @@
+export const minLength = value => (value && value.length < 8 ? ` Must be 8 characters or more` : undefined);
+
+export const required = value => (value ? undefined : 'Required');
```

Comment:
Make a `isEmail` validation here.
Then make a `userValidation` and `passwordValidation` validators that combine the previous functions and export them.